src/components/Timer/Timer.module.css

```
@@ -120,3 +120,20 @@
     padding: 1px 3px;
   }
 }
+
+@media (max-width: 360px) {
+  .btns {
+    display: grid;
+    grid-template-columns: auto auto;
+  }
+
+  .btns button {
+    padding: 1px 1px;
+  }
+}
+
+@media (max-width: 530px) {
+  .btnDiv{
+    display: none;
+  }
+}
```

> > @Nahiyan-16 I think the update has not been pushed yet. Could you please check it?
> 
> I'm not too sure what changes you are requesting. Can you please clarify?

Sure! If Prettier has been successfully executed, the code would look like this.
```suggestion
@media (max-width: 530px) {
  .btnDiv {
    display: none;
  }
}
```